packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/AdminsAndModerators.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../../../../styles/shared';
+
+.AdminsAndModerators {
```

Comment:
lets try to use the new `CommunityManagementLayout`, then we can cleanup this CSS file

Reply:
edit: just saw you already implemented that, so we can remove some of this unused css